--- scripts/mysqlhotcopy.sh	2010-12-03 18:58:26.000000000 +0100
+++ scripts/mysqlhotcopy.sh.oden	2010-12-27 13:42:50.919921524 +0100
@@ -790,7 +790,24 @@ sub get_list_of_tables {
         } || [];
     warn "Unable to retrieve list of tables in $db: $@" if $@;
 
-    return (map { $_->[0] } @$tables);
+    my @ignore_tables = ();
+
+    # Ignore tables for the mysql database
+    if ($db eq 'mysql') {
+        @ignore_tables = qw(general_log slow_log schema apply_status);
+    }
+
+    my @res = ();
+    if ($#ignore_tables > 1) {
+       my @tmp = (map { $_->[0] } @$tables);
+       for my $t (@tmp) {
+           push(@res, $t) if not exists { map { $_=>1 } @ignore_tables }->{$t};
+       }
+    } else {
+       @res = (map { $_->[0] } @$tables);
+    }
+
+    return @res;
 }
 
 sub quote_names {
