--- mysql-test/t/disabled.def	2011-09-09 17:56:40.000000000 +0200
+++ mysql-test/t/disabled.def.oden	2011-09-16 14:11:43.374835601 +0200
@@ -17,3 +17,8 @@
 archive-big              : Bug#11817185 2011-03-10 Anitha Disabled since this leads to timeout on Solaris Sparc
 log_tables-big           : Bug#11756699 2010-11-15 mattiasj report already exists
 mysql_embedded           : Bug#12561297 2011-05-14 Anitha Dependent on PB2 changes - eventum#41836
+#
+outfile_loaddata         : bug#46895 code wrong, expected results wrong too
+sys_vars.plugin_dir_basic : bug#52223 fails for lib64 library directory
+perfschema.binlog_mix    : bug#59091 fails with openssl
+perfschema.binlog_row    : bug#59091 fails with openssl
